_posts/2021-04-12-ballista-donation.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Ballista: A Distributed Scheduler for Apache Arrow"
+description: "We are excited to announce that Ballista has been donated to the Apache Arrow project. Ballista is a distributed scheduler for the Rust implementation of Apache Arrow."
+date: "2021-04-12 00:00:00 -0600"
+author: agrove
+categories: [application]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+We are excited to announce that [Ballista](https://github.com/apache/arrow/tree/master/rust/ballista) has been donated 
+to the Apache Arrow project. 
+
+Ballista is a distributed compute platform primarily implemented in Rust, and powered by Apache Arrow. It is built
+on an architecture that allows other programming languages (such as Python, C++, and Java) to be supported as
```

Comment:
I wonder if we want to callout the use of DataFusion here (and in the list of foundational technologies) as part of a broader "marketing campaign" for DataFusion) ?

Reply:
Excellent point! Feel free to push to this PR or I can add something this evening.